packages/react-components/src/components/utils/available-views-component.tsx

```diff
@@ -0,0 +1,107 @@
+import { ListRowProps, AutoSizer, List } from 'react-virtualized';

```

Comment:
This is a modified copy of `trace-explorer-views-widget.tsx`. Do you plan to refactor the code in a separate PR to optimize code re-use and remove code duplication? Please let me know.

Reply:
Yes there is a PR that is in draft state for this refactoring. Once this one is approve, I will put that as ready for review.